clustered/common/src/main/java/org/ehcache/clustered/common/ServerStoreCompatibility.java

```diff
@@ -42,27 +42,28 @@ public void verify(ServerStoreConfiguration serverConfiguration, ServerStoreConf
 
     PoolAllocation serverPoolAllocation = serverConfiguration.getPoolAllocation();
     PoolAllocation clientPoolAllocation = clientConfiguration.getPoolAllocation();
```

Comment:
yes that makes sense.   I will try to refactor code into instance methods.  This should help clean it up a bit too.